08-rails/rails-forms.md

```diff
@@ -32,43 +32,43 @@ Within the `form_tag` block, additional form helpers can be used to create input
 it expects is the HTML name attribute. The second is the default value of the input. Additional HTML options can be passed in a hash. For example:
 
 ```erb
-<%= text_field_tag :pet[type], "elephant", class: "pets" %>
+<%= text_field_tag :book[author], "testing", class: "some-class" %>
 ```
 
 Results in:
 
 ```html
-<input type="text" name="pet[type]" id="pet_type" value="elephant" class="pets" />
+<input type="text" name="book[author]" id="book_author" value="testing" class="some-class" />
 ```
 
 #### `submit_tag`
 As the name implies, the `submit_tag` _view helper_ generates a submit button for a form created with `form_tag`. It accepts two parameters, both optional. The first is the text that should appear in the button (defaults to "Submit"), and the second is a hash of HTML attributes:
 
 ```erb
-<%= submit_tag "Save Pet", class: "pet_button" %>
+<%= submit_tag "Save Book", class: "book-button" %>
 ```
 
 Results in:
 
 ```html
-<input type="submit" name="commit" value="Save Pet" class="pet_button" />
+<input type="submit" name="commit" value="Save Book" class="book-button" />
 ```
 
 Many, many other _view helpers_ are available to help build any type of form or input. Look at the [form helper docs](http://api.rubyonrails.org/classes/ActionView/Helpers/FormTagHelper.html) for complete documentation.
 
 ## form_for
 `form_for` is similar to `form_tag`, except it is meant to be used in conjunction with an Active Record model.
 
-The only required argument is takes is any ActiveRecord object. Rails will make assumptions about how to structure the form's method and action based on RESTful conventions (POST to '/products' for creating new records and PUT/PATCH to '/products/:id' for updating existing records, etc). Check it:
+The only required argument is takes is any ActiveRecord object. Rails will make assumptions about how to structure the form's method and action based on RESTful conventions (POST to '/books' for creating new records and PUT/PATCH to '/books/:id' for updating existing records, etc). Check it:
```

Comment:
The only requirement **is** takes --> The only argument `form_for` takes is any ActiveRecord object.